client/components/Scenario/ContentSlide.jsx

```diff
@@ -160,6 +160,15 @@ class ContentSlide extends React.Component {
             ? ` (${pending.length} required responses are not complete)`
             : '';
 
+        let skipButtonContent = skipButton;
+
+        if (isLastSlide) {
```

Comment:
I'm realizing we don't need to the index to determine the last slide with this PR, it's saved on the slide itself! That might be worth refactoring in this PR. This way we can make `Scenario/index.jsx` simpler.

Reply:
We use the `index` as a `key`, otherwise React complains. When I remove it: 

```
Warning: Each child in a list should have a unique "key" prop.
```